README.md

```diff
@@ -28,12 +28,12 @@ https://polymerlabs.github.io/arcs-live/shell/apps/web/.
 
 ## Install
 
-Arcs is developed with a recent version of Node (v9.4.0 at the time of this
+Arcs is developed with a recent version of Node (v10.0.0 at the time of this
```

Comment:
Is it 10.0.0 or 10.12.0? Just now on my machine:

```wkorman@lemur:~$ nvm install 10
Downloading and installing node v10.12.0...
```

As long as we tested with 10.0.0 it's fine.